service/history/engine/engineimpl/notify_tasks.go

```diff
@@ -112,5 +112,5 @@ func hydrateReplicationTask(
 		history.Find(info.NewRunBranchToken, constants.FirstEventID),
 	)
 
-	return hydrator.Hydrate(context.Background(), info)
+	return hydrator.Hydrate(context.Background(), task)
```

Comment:
why do we even need `info` object in this func?

Reply:
it's used by hydrator